devnet/devnet_test.go

```diff
@@ -44,26 +48,6 @@ func TestDevnet_Accounts(t *testing.T) {
 	}
 }
 
-// TestDevnet_FeeToken tests the FeeToken function of the Devnet struct.
```

Comment:
Why was this removed? I think we can add it back in without breaking tests

Reply:
I removed this tests because the[ new devnet doesn't have the "/fee_token" endpoint anymore](https://github.com/0xSpaceShard/starknet-devnet-rs/blob/main/crates/starknet-devnet-server/src/server.rs#L34). That being said, I believe the "Mint" and "TestCall" tests are enough to prove the token existing, address correctness, and functionalities.